client/src/ui/diagram.html

```diff
@@ -78,143 +67,109 @@ <h5 class="modal-title" id="modal-title">Modal title</h5>
-            var samlRequestBtn = dom.querySelector(query);
-            samlRequestBtn.disabled=false;
-            samlRequestBtn.classList.remove("btn-secondary");
-            samlRequestBtn.add("btn-primary");
-            buttonClick(query,`SAML Request`,req);
-            blinking(query)
-        }
-
-        if (!res.includes("SAML_RESPONSE")){
-            clearInterval(timer);
-            var query='#saml-res-btn';
-            var samlResponsetBtn = dom.querySelector(query);
-            samlResponsetBtn.disabled=false;
-            samlResponsetBtn.classList.remove("btn-secondary");
-            samlResponsetBtn.add("btn-primary");
-            buttonClick(query,`SAML Response`,req);
-            blinking('#saml-res-btn')
-        }
-
-        return dom.innerHTML;
-    }
-
-
-    function addConfig(req,res){
-        if (!req.includes("SAML_REQUEST")){
-            var card = document.getElementById("samlsso-card");
-            card.classList.add("alert-primary");
-            var reqBtn = document.getElementById("req-btn");
-            reqBtn.disabled=false;
-            reqBtn.classList.remove("btn-secondary");
-            reqBtn.classList.add("btn-primary");
-            var query='#req-btn';
-            buttonClick(query,`SAML Request`,req);
-            blinking(query)
-        }
-
-        if (!res.includes("SAML_RESPONSE")){
-            clearInterval(timer);
-            var resBtn = document.getElementById("res-btn");
-            resBtn.disabled=false;
-            resBtn.classList.remove("btn-secondary");
-            resBtn.classList.add("btn-primary");
-            var query='#res-btn';
-            buttonClick(query,`SAML Response`,req);
-            blinking(query)
-        }
-    }
-
-
     $(document).ready(function () {
         const vscode = acquireVsCodeApi();
-        var filepath = '{myfilepath}';
-        var text = `{myXML}`;
+        const JOB_TYPE_SAML_SSO = "samlsso";
+        const JOB_TYPE_SAML_OPENID = "openid";
+        let filepath = '{myfilepath}';
+        let text = `{myXML}`;
         xmlDoc = $.parseXML(text);
         $xml = $(xmlDoc);
-        x = document.getElementById("item_1");
+        let inboundAuthElem = document.getElementById("inbound-auth-div");
         y = document.getElementById("item_input");
-        var samlRequest= `{SAML_REQUEST}`;
-        var samlResponse= `{SAML_RESPONSE}`;
-
-        $xml.find("InboundAuthenticationConfig").find("InboundAuthenticationRequestConfigs").find("InboundAuthenticationRequestConfig").each(function () {
-
-            var $row = $(this),
-                name = $row.find("column1").text(),
+        let samlRequest = `{SAML_REQUEST}`;
+        let samlResponse = `{SAML_RESPONSE}`;
+        let oidcAuthzRequest = `{OIDC_AUTHZ_REQUEST}`;
+        let oidcAuthzResponse = `{OIDC_AUTHZ_RESPONSE}`;
+        let oidcTokenRequest = `{OIDC_TOKEN_REQUEST}`;
+        let oidcTokenResponse = `{OIDC_TOKEN_RESPONSE}`;
+        let issamlSSO = true;
```

Comment:
We are not updating these values again right? Shall we convert them to `const`

Reply:
Fixed with https://github.com/wso2-extensions/identity-tools-plugin-vscode/pull/7